.gitignore

```diff
@@ -103,6 +103,7 @@ linux-coverage-build
 linux-build
 win32-build
 qa/pull-tester/tests_config.py
+qa/pull-tester/ltc-scrypt
```

Comment:
is this still needed now that we install from archive?

Reply:
it is not. i have removed.